src/ServiceProvider.php

```diff
@@ -24,14 +24,16 @@ public function register(): void
         $this->app->singleton(ClientContract::class, static function (): Client {
             $apiKey = config('openai.api_key');
             $organization = config('openai.organization');
+            $baseUri = config('openai.base_uri', 'api.openai.com/v1');
 
-            if (! is_string($apiKey) || ($organization !== null && ! is_string($organization))) {
+            if (! is_string($apiKey) || ! is_string($baseUri) || ($organization !== null && ! is_string($organization))) {
```

Comment:
Just a exception for all cases?  Maybe we can reconsider this..

Reply:
![image](https://github.com/openai-php/laravel/assets/17726941/3f98fa0e-a8ef-4c66-8cf4-4168a897beea)

Without this condition `! is_string($baseUri)` it cannot pass phpstan. Do you have better solution?

Reply:
I didn't mean remove !is_string condition, instead of this, you can use different if block and exception for baseUrl